ibm/mas_devops/roles/gencfg_uds/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+# 1. Check for required facts
+# -----------------------------------------------------------------------------
+- name: "Fail if uds_endpoint_url has not been provided"
+  assert:
+    that: uds_endpoint_url is defined and uds_endpoint_url|length == 0
```

Comment:
it should be a check for `length != 0` and not `length == 0`. Same for the ones below